### PR TITLE
Adding new functionality to sign out button

### PR DESCRIPTION
**Pull request by mlcvijetinovic**
Implemented new functionality to sign out button,and now we sign out from
seassion, not google account.We achived this,by passed global auth object, so it's visible
after redirection.All this is achived with onload call function.Also
added new i18n implementation on login page.

### CODE REVIEW

**Comment by stanislavgrujic** [comment]
Why are all files reported as completely changed?

**Comment by mlcvijetinovic** [comment]
I do not know the real reason why it changed everything.Maybe because, java code formatter is set to perform action on save? Updates are made only at the specified line of code, not at all.I can write comment for each line of code that is changed.

**Comment by mlcvijetinovic** [comment]
Or to delete pull request,and execute commit and push again,but to pay attention that the file reports shows only the corrected code, not all.

**Comment by stanislavgrujic** [comment]
I'd look at the line feeds configured in the formatter. It could happen if you're developing on Linux and Windows machines. Please check it and find how to overcome it.

**Comment by mlcvijetinovic** [comment]
I have made changes that you advised me,related to different line feed ubuntu windows in eclipse setup.

File: src/main/resources/messages_de.properties [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
-user.delete.member=Entfernen
-user.add.member=Neues Mitglied
-user.add.team=Neues Team
-
-team.save.button=Sparen
-team.close.button=Schlie�en
-team.confirm.delete.button=L�schen best�tigen
+login.registration=Teamodoro Registrierung
+login.sign.out=Ausloggen
+login.registration.event=Laden...
```
Comment (mlcvijetinovic) [inline src/main/resources/messages_de.properties]: Added this for internationalization.When user click on register button,user receives a message that redirects to the next page.

File: src/main/resources/messages_de.properties [hunk 2]
```diff
@@ -1,23 +1,24 @@
...
+confirm=Best�tigen
+
+user.welcome.message=Willkommen
```
Comment (mlcvijetinovic) [inline src/main/resources/messages_de.properties]: Added this for internationalization.When user click on sing in button,user receives welcome message until redirected.

**Pull request closed without merging**